<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search_by_name_widget</class>
 <widget class="QWidget" name="search_by_name_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="1" column="2">
      <widget class="QPushButton" name="button_line_edit_clear">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
   background-color: white;
   color: black;
   border: 2px solid black ; /* Black */
	text-align: center;
   border-radius:  10px;
	font-size: 20pt;
	padding-left: 10px;
	padding-right: 10px;
}

QPushButton:hover{
	background-color: rgb(225, 225, 225);
}
QPushButton:pressed{
	background-color: rgb(195, 195, 195);
}
</string>
       </property>
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="go_main_menu_button">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
   background-color: white;
   color: black;
   border: 2px solid black ; /* Black */
	text-align: center;
   border-radius:  10px;
	font-size: 20pt;
	padding-left: 10px;
	padding-right: 10px;
}

QPushButton:hover{
	background-color: rgb(225, 225, 225);
}
QPushButton:pressed{
	background-color: rgb(195, 195, 195);
}
</string>
       </property>
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
   background-color: white;
   color: black;
    border: 2px solid black ; /* Black */
	text-align: center;
    border-radius:  10px;
	font-size: 20pt;
}</string>
       </property>
       <property name="text">
        <string>Поиск по названию</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>876</width>
           <height>443</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="3">
           <layout class="QGridLayout" name="gridLayout_2"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
   background-color: white;
   color: black;
   border: 2px solid black ; /* Black */
	text-align: center;
   border-radius:  10px;
	font-size: 20pt;
}
QPushButton:hover{
	background-color: rgb(225, 225, 225);
}
QPushButton:pressed{
	background-color: rgb(195, 195, 195);
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="layout_for_keyboard_widget">
        <property name="topMargin">
         <number>10</number>
        </property>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="open_close_keyboad_button">
          <property name="styleSheet">
           <string notr="true">	padding-left: 10px;
	padding-right: 10px;</string>
          </property>
          <property name="text">
           <string>Закрыть клавиатуру</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
